src/Command/ContentHubUpgradeStart.php

```diff
@@ -716,4 +769,57 @@ protected function executeValidateInterestListDiff(PlatformInterface $platform,
     }
   }
 
+  /**
+   * Helper method to send logs to amplitude.
+   *
+   * @param string $event_name
+   *   Event to send to amplitude.
+   * @param int $step
+   *   Upgrade step user currently is on.
+   * @param string $message
+   *   Message to be shown for this step.
+   */
+  private function sendLogsToAmplitude(string $event_name, int $step, string $message) {
+    $this
+      ->amplitudeClient
+      ->logEvent($event_name, array_merge($this->userDetails, [
+        'step' => $step,
+        'message' => $message,
+      ]));
+  }
+
+  /**
+   * Initializes Amplitude Client.
+   *
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *   Output stream.
+   */
+  private function initializeAmplitudeClient(OutputInterface $output) {
```

Comment:
I'd put this on a `Trait` so we could use it in other commands we would want to capture information from.

Reply:
Done. Moved to a trait.